Author: Sven Joachim <svenjoac@gmx.de>
Description: Fix copy/paste error in configure.in
 Fix the --disable-root-access and --disableroot-environ configure
 options.  Due to a copy/paste error, the latter performs the actions
 of the former, while the --disable-root-access option has no effect
 at all.
Forwarded: https://lists.gnu.org/archive/html/bug-ncurses/2023-04/msg00003.html
Last-Update: 2023-05-01

---
 configure.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/configure.in
+++ b/configure.in
@@ -868,7 +868,7 @@ AC_MSG_RESULT($with_root_environ)
 test "x$with_root_environ" = xyes && AC_DEFINE(USE_ROOT_ENVIRON,1,[Define to 1 if root is allowed to use ncurses environment])
 
 AC_MSG_CHECKING(if you want to permit setuid programs to access all files)
-AC_ARG_ENABLE(root-environ,
+AC_ARG_ENABLE(root-access,
 	[  --disable-root-access   restrict file-access when running setuid],
 	[with_root_access=$enableval],
 	[with_root_access=yes])
